feature/experimental/gnoi/packet_link_qualification/tests/packet_link_qualification_test/packet_link_qualification_test.go

```diff
@@ -231,22 +191,10 @@ func TestNonexistingID(t *testing.T) {
 
 	deleteResp, err := gnoiClient1.LinkQualification().Delete(context.Background(), &plqpb.DeleteRequest{Ids: []string{id}})
 
-	// TODO: Remove fakeResp and uncomment err checking if deleteResp is received from DUT.
-	// if err != nil {
-	// 	t.Fatalf("Failed to handle gnoi LinkQualification().Get(): %v", err)
-	// }
 	t.Logf("LinkQualification().Get(): %v, err: %v", getResp, err)
-
-	fakeDeleteResp := &plqpb.DeleteResponse{
```

Comment:
presume two other instances of fakeResp in lines 373 and 402 can exist for now?

Reply:
Yes, I will replace them once the test passed without fake response.